ADMIN-PORT.md

```diff
@@ -0,0 +1,44 @@
+---
+RFC: 3
+Status: Proposed
+---
+
+# ADMIN PORT RFC
+
+## Abstract
+
+In Valkey.conf, Add a new config parameter: "admin-port" to allow internal administrators in a company or special users from public internet to connect to the Valkey server to execute some special commands or load specfic modules.
+
+
+## Motivation
+
+For adminstrators, they may want to execute some special commands in the server, and other general clients are not allowed to run these kinds of commands.
+We want to let adminstrator connect to this admin port to do some special things. 
+
+Beside above mentioned, we could implement the following features in the future:
+1. The connections through admin port are not limited to maxclient number, we can always guarantee they can connect to the server.
+2. Some status check commands could be sent and receied through admin port, we can do some special logic to make sure they have opportunity to be handle first
```

Comment:
I think it would be useful to enumerate these here. Maybe everything that is not a data command? Reminds me a bit of this check: https://github.com/valkey-io/valkey/blob/5166d489da1bbf49643dd6b44cd2593e21c09dc3/src/server.c#L4061.